lib/membrane/video_compositor/scene/transformation.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Membrane.VideoCompositor.Scene.Transformation do
+  @moduledoc """
+  Behaviour representing single input, single output transformations of objects.
```

Comment:
```suggestion
  Behaviour representing single input, single output transformation of objects.
```

Reply:
And are you sure we can call it a "behaviour"? In Elixir term "behaviour" has a very strict meaning.

Reply:
As we aren't sure if Transformation will end up as a behaviour, i'll change it to the module